addons/im_livechat/static/tests/mock_server/mock_models/res_lang.js

```diff
@@ -0,0 +1,5 @@
+import { models } from "@web/../tests/web_test_helpers";
+
+export class ResLang extends models.ServerModel {
+    _name = "res.lang";
+}
```

Comment:
Yes, in legacy mock server we could invent some models that were not defined. Hoot mock server wants these models to exist so it has to be introduced.

Also the models of mock server are scoped by suite with `defineModels()`. As to avoid taking the risk to break existing `web/` tests (that could result in wasting time for be to convert `mail` tests), I cautiously put required models where I immediately needed.